test/JLSOFile.jl

```diff
@@ -55,3 +62,27 @@ end
         @test Pkg.TOML.parsefile(joinpath(d, "Manifest.toml")) == jlso.manifest
     end
 end
+
+@testset "keys/haskey" begin
+    jlso = JLSOFile(:string => datas[:String])
+    @test collect(keys(jlso1)) == [:string]
+    @test haskey(jlso, :string)
+    @test !haskey(jlso, :other)
+end
+
+@testset "get/get!" begin
```

Comment:
wasn't sure which file made most sense for these methods and tests... so let me know if this isn't the right call :)

Reply:
This seems fine. The methods exists in the correponding `src` file anyways.